aptos-move/framework/supra-framework/sources/automation_registry.move

```diff
@@ -4,13 +4,12 @@
 module supra_framework::automation_registry {
 
     use std::signer;
-    use std::vector;
-
-    use supra_std::enumerable_map::{Self, EnumerableMap};
+    use supra_framework::event;
+    use supra_framework::automation_registry_state::AutomationTaskMetaData;
+    use supra_framework::automation_registry_state;
```

Comment:
Just a suggestion: It would be cleaner to import as use `supra_framework::automation_registry_state::{Self, AutomationTaskMetaData}` instead of using two separate lines for imports